vortex-fastlanes/src/lib.rs

```diff
@@ -3,7 +3,8 @@
 
 pub use bitpacking::*;
 pub use delta::*;
-use linkme::distributed_slice;
+pub use downcast::*;
+pub use linkme::distributed_slice;
```

Comment:
```suggestion
use linkme::distributed_slice;
```
This shouldn’t be pub